src/Veldrid.Utilities/FastParse.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+
+namespace Veldrid.Utilities
+{
+    public static class FastParse
```

Comment:
I'd like to see if we can remove this file altogether, but if it needs to stay, it should be `internal`.

Reply:
Can be removed with the netstandard2.1 target